modules/ROOT/pages/dataweave-cookbook-flatten-arrays.adoc

```diff
@@ -0,0 +1,316 @@
+= Flatten Elements of Arrays
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
+
+DataWeave can flatten subarrays of an array and collections of key-value pairs
```

Comment:
I wonder if this should be worded to address this as a user goal or task, and why?
Something like, "You can do this using DataWeave, and here is why you might want or need to do it."

Reply:
Also, do you think that this existing intro sentence prepares your audience for the three specific subtopics within? 
It can be a good idea to make the mentions of subtopics in the intro text the same as the titles the audience finds below.

Reply:
It's a developer audience, so I think it's sufficient.

Reply:
Their goal is to flatten the subarrays and collections of key-value pairs...